nilearn/input_data/nifti_masker.py

```diff
@@ -189,6 +192,24 @@ def __init__(self, mask_img=None, sessions=None, smoothing_fwhm=None,
         self.memory_level = memory_level
         self.verbose = verbose
 
+        self._shelving = False
+
+    def _deactivate_shelving(self):
+        self._shelving = True
+
+    def _activate_shelving(self):
+        """Activate shelving.
+
+        When shelving is activated, the transform method will shelve the result
+        on disk and return an object region_signals with method get:
+        region_signals.get() yields the maked array.
+        Useful to parallelize the masking of records before entering a
+        sequential pipeline."""
+        self._shelving = True
```

Comment:
> I agree. We only need a _set_shelving one.

We don't need a setter. This is not java. We just change the value of the
attribute directly. The methods do nothing else.